app/controllers/blog.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+const {Controller, computed, inject} = Ember;
+
+export default Controller.extend({
+	application: inject.controller(),
+	article: inject.controller(),
+	wikiVariables: inject.service(),
+
+	commentsPage: computed.alias('application.commentsPage'),
+	mainPageTitle: computed.reads('wikiVariables.mainPageTitle'),
+	siteName: computed.reads('wikiVariables.siteName'),
```

Comment:
I see a lot thos attributes being copied from other conteoller. Maybe it's a time to create a mixin for them?